### PR TITLE
Push Environments

### PR DESCRIPTION
**Pull request by palanc**
1. Update existing push env to use cartesian action space
2. Add new pushing envs
3. All pushing envs use robotiq gripper. See additional PR here for gripper updates: https://github.com/vikashplus/robotiq_sim/pull/1

### CODE REVIEW

File: robohive/envs/arms/franka/assets/franka_ycb_v0.config [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     # device1: sensors, actuators
     'franka':{
-        'interface': {'type': 'franka', 'ip_address':'169.254.163.91'},
+        'interface': {'type': 'franka', 'ip_address':'172.16.0.1', 'gain_scale': 1.5},
```
Comment (vikashplus) [inline robohive/envs/arms/franka/assets/franka_ycb_v0.config:4]: Scale can only be a max of 1.0

File: robohive/envs/arms/franka/assets/franka_ycb_v0.xml [hunk 2]
```diff
@@ -10,12 +10,12 @@
 
     <size njmax='1000' nconmax='1000'/>
 
+
     <include file="../../../../simhive/scene_sim/topfloor_scene.xml"/>
     <include file="../../../../simhive/franka_sim/assets/assets.xml"/>
-    <include file="../../../../simhive/franka_sim/assets/gripper_assets.xml"/>
+    <include file="../../../../simhive/robotiq_sim/assets/assets.xml"/>
```
Comment (vikashplus) [inline robohive/envs/arms/franka/assets/franka_ycb_v0.xml:16]: Note that this is franka robot. Adding RobotiQ will cause confusion. We should create a new file for this and name it appropriately.

File: robohive/envs/arms/push_base_v0.py [hunk 3]
```diff
@@ -25,6 +30,27 @@ class PushBaseV0(env_base.MujocoEnv):
 
     def __init__(self, model_path, obsd_model_path=None, seed=None, **kwargs):
```
Comment (vikashplus) [inline robohive/envs/arms/push_base_v0.py:31]: Currently it looks like, the original env is being replaced by this new IK based env. Right? I think the right way to implement this is to 
1. define a new [act mode](https://github.com/vikashplus/robohive/blob/main/robohive/robot/robot.py#LL47C48-L47C52 ) called IK
2. Once the robot receives `ctrl_desired` in the step, it relies on the [process-actuator](https://github.com/vikashplus/robohive/blob/main/robohive/robot/robot.py#L673) call to do the translation from ee space to jnt space.
3. I also think the [action space definitions](https://github.com/vikashplus/robohive/blob/main/robohive/envs/env_base.py#LL95C9-L98C1) needs to be updated if its IK space

In summary, landing this in a generic way seems to require a rather larger refactor (possibly involving a refactor of the robot class). Let me know if should discuss this before going down this route.

**Review by vikashplus — COMMENTED** [review]
This seems like a larger refactor. See comments inline
